frontend/src/js/small-tab-navigation/HoverNavigatable.tsx

```diff
@@ -28,7 +28,6 @@ const Root = styled("div")<{
     return "inherit";
   }};
   position: relative;
-  border-radius: ${({ theme }) => theme.borderRadius};
```

Comment:
Why this change? This makes the background bleed out the container at least for the editor